OpenKeychain/src/main/java/org/sufficientlysecure/keychain/keyimport/CloudSearch.java

```diff
@@ -52,6 +52,8 @@
             servers.add(FacebookKeyserverClient.getInstance());
         }
 
+        servers.add(WebKeyDirectoryClient.getInstance());
```

Comment:
This looks like it should also be gated with `cloudPrefs.searchXXX` right?

Reply:
I've modified the source so in case a non-supported name is searched for WebKeyDirectoryClient just returns an empty list so no `cloudPrefs.searchWKD` is needed... I think.

Reply:
I think a setting would still be nice, if only for discoverability of the fetaure.